AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Commands/AWSSDKSwiftCLI/Subcommands/GeneratePackageManifest.swift

```diff
@@ -35,14 +35,18 @@ struct GeneratePackageManifestCommand: ParsableCommand {
     @Flag(help: "If the package manifest should exclude runtime tests.")
     var excludeRuntimeTests = false
 
+    @Flag(help: "If the package manifest should be configured for a preview build.")
+    var previewBuild = false
+
```

Comment:
The CLI flag that is being added.  Note it defaults to false, which renders a manifest with current behavior.